eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,7 +1,8 @@
 variables:
   DocWardenVersion: '0.5.0'
-  DotNetCoreSDKVersion: '3.1.100'
-  DotNetCoreRuntimeVersion: '2.1.10'
+  DotNetCoreSDKVersion: '3.1.201'
+  DotNetCoreRuntimeVersion2.1: '2.1.10'
```

Comment:
While we are updating can you also update the latest patch version of 2.1? Looks like 2.1.17 https://dotnet.microsoft.com/download/dotnet-core/2.1.